server/lib/scraper.js

```diff
@@ -248,6 +248,39 @@ function club(id) {
   })
 }
 
+function clubPreview(id) {
+  return new Promise((res, rej) => {
+    osmosis
+      .get(
+        resolve(
+          host,
+          `/cgi-bin/WebObjects/nuLigaTTCH.woa/wa/clubInfoDisplay?club=${id}`
```

Comment:
This is requesting the same click-tt page as the `club` call - wouldn't it make more sense to merge the two calls into one, and return all the information in a single response?